src/index.ts

```diff
@@ -9,6 +9,8 @@ import settings from "./settings.js";
 import { helper } from "./chatbot.js";
 import { QueueEntry } from "./extensions-api/queue-entry.js";
 import { Chatter, Responder } from "./extensions-api/command.js";
+import "./i18n.js";
```

Comment:
I would like this to be `await import("./i18n.js");`, because otherwise
```
Initializing i18next...
Language: en
```
would be printed above the banner.

Reply:
Good catch. I also didn't mean to keep printing the language there, so I've removed that.